src/article-editor.jsx

```diff
@@ -7,11 +7,12 @@ var Editor = require("./editor.jsx");
 var EnabledFeatures = require("./enabled-features.jsx");
 var JsonEditor = require("./json-editor.jsx");
 var Renderer = require("./renderer.jsx");
+var Util = require("./util.js");
 
 var rendererProps = React.PropTypes.shape({
     content: React.PropTypes.string,
     widgets: React.PropTypes.object,
-    images: React.PropTypes.object,
+    images: React.PropTypes.object
```

Comment:
That makes sense.  All of the commas should be back in place.  It'd be nice to have a .jshintrc or something like that.